spec/board_presenter_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "board_presenter"
+require "board"
+
+describe BoardPresenter do
+    before(:each) do
+        @board_presenter = BoardPresenter.new
+    end
+
+    describe "#display_board" do
+        it "renders empty board if there are no player marks" do
+            board = Board.new
+
+            board.moves.length == 0
```

Comment:
`==` is an equality check. Not a variable assignment. What's the purpose of this line?

Reply:
While the 'moves' array is empty, the board should also be empty. So future tests would have board moves, and the board should correlate with the moves that have been made.

Reply:
Ok. What's the purpose of this line in the test though? It's not assigning anything or asserting anything.

Reply:
To better put things into perspective, if you delete this line. Your tests will still pass.

Reply:
Absolutely! That was for my own sanity check. I have deleted that line.